### ISSUE TITLE
Deleting Params is too difficult

### ISSUE THREAD

**Issue by Zethson** [laminlabs/lamindb#2254]
### Add a description

I needed to delete some `Transform` that had `Params` (`Run`) and therefore also needed to delete the `ParamValues`. It was not obvious to me that they are in `lamindb.core`. Moreover, it was even less obvious that I need to delete `RunParamValue` which I can only access if I directly access `lnschema_core`. No user will ever get that.

I was still annoyed that https://github.com/laminlabs/lamindb/issues/1547 is not yet implemented. Maybe I'll do it at some point.

**Comment by falexwolf** [comment]
Yes, so my plan was adding `run.params.remove_values()` in analogy to what I did for features a week ago:

- https://github.com/laminlabs/lamindb/pull/2218

**Comment by falexwolf** [comment]
This is outdated.
